File: client/directScripts/AWGpollError.py
import write
import query
import sys
AWGadd = "TCPIP0::localhost::inst1::INSTR"


def err():
	# AWGrun.run
	Command = ":SYST:ERR?"
	return query.query(AWGadd,Command)

	
if __name__ == "__main__":
	if len(sys.argv) == 1:		# run
		result = err()
		print(result)
	else:
		print("err expected 0 arguments, got "+str(len(sys.argv)-1)+".")
		sys.exit(1)

File: client/simpleUpload&Capture/AWGcontrols/AWGpollError.m
function errString = AWGpollError()
%AWGREFCLOCKQUERY Summary of this function goes here



AWGadd = "TCPIP0::localhost::inst1::INSTR";
Command = ":SYST:ERR?";

% in the form of ">python (python_command) (device) (device_command)"
cmdStr = "python query.py " + AWGadd + " " + Command;

[status,cmdOut] = system(cmdStr);
if status==2
    warning("file not found")
elseif status==0
	errString = cmdOut;
end